frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -559,7 +560,7 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                 }
 
                 const uniqueUserSeries: EventsNode = {
-                    event: '$pageview',
+                    event: featureFlags[FEATURE_FLAGS.WEB_ANALYTICS_FOR_MOBILE] ? '$screen' : '$pageview',
```

Comment:
I'm curious, what's your long term idea for this? What if someone wants to support both mobile and web? This will most likely occur, right?

Is the idea here that they should set up separate projects to handle that - one of them tracking `$pageview`s and the other `$screen`s?

Reply:
We also want to show web analytics for a specific website, rather than across all of your sites, so I think we could combine it with that. If the "site" is a mobile app, then put it in mobile app mode.

Reply:
Yeah, that makes sense. This would be pretty easy to add as a toggle at the project level then :)